packages/jobs/lib/execution/operations/output.ts

```diff
@@ -33,25 +33,25 @@ export async function handleSuccess({ taskId, nangoProps, output }: { taskId: st
     }
 }
 
-export async function handleError({
-    taskId,
-    nangoProps,
-    error
-}: {
-    taskId: string;
-    nangoProps: NangoProps;
-    error: {
-        type: string;
-        payload: Record<string, unknown>;
-        status: number;
-    };
-}): Promise<void> {
+export async function handleError({ taskId, nangoProps, error }: { taskId: string; nangoProps: NangoProps; error: RunnerOutputError }): Promise<void> {
     const formattedError = toNangoError({
         err: error,
         defaultErrorType: `${nangoProps.scriptType}_script_failure`,
         scriptName: nangoProps.syncConfig.sync_name
     });
 
+    if (error.type === 'script_aborted') {
```

Comment:
I am not a huge fan of NangoError.type is a `string` and not a union type. I tried to refactor it but it is basically opening the pandora box so I gave up (for now) :(